action.yml

```
@@ -0,0 +1,22 @@
+---
+author: Cyber and Infrastructure Security Agency
```

Use our official agency name:
```suggestion
author: Cybersecurity and Infrastructure Security Agency
```